src/styles.css

```diff
@@ -59,5 +59,5 @@
 
 .chat-gpt-container #answer code {
   white-space: pre-wrap;
-  word-break: break-all;
+  word-break: break-word;
```

Comment:
Why?

Reply:
> Why?

'break-all' breaks words, so it might be nicer to use 'break-word'

Reply:
> Why?

Here's an example

with `break-all`

![image](https://user-images.githubusercontent.com/91773459/206675727-25561472-c8c8-4357-8419-7a3935d07bc3.png)

with `break-word`

![image](https://user-images.githubusercontent.com/91773459/206676109-361d9e37-c24c-4420-a89c-a9ff62adc907.png)